macros/Compat.ejs

```diff
@@ -71,22 +70,40 @@ const mobileBrowsers = {
   safari_ios: 'iOS Safari',
 }
 
+const webExtBrowserNames = {
+  chrome: "Chrome",
+  edge: "Edge",
+  firefox: "Firefox",
+  firefox_android: s_firefox_android,
+  opera: "Opera"
+};
+
 var notesArray = [];
 
 /*
 Write the table header.
 
-`browserPlatformType` is either "mobile" or "desktop"
-`browserNames` is either the "mobileBrowsers" or "desktopBrowsers" object
+`browserPlatformType` is either "mobile", "desktop" or "webextensions"
+`browserNames` is either the "mobileBrowsers", "desktopBrowsers" or "webExtBrowserNames" object
```

Comment:
I refactored this to have a browser map and to only pass `browserPlatformType` to the various functions.